Dockerfile

```diff
@@ -8,5 +8,8 @@ RUN pip3 install pipenv
 RUN pipenv requirements > requirements.txt
 RUN pip3 install -r requirements.txt --target "${LAMBDA_TASK_ROOT}"
 
-# Default handler. See README for how to override to a different handler.
+# Set environment variables required by DuckDB
+ENV HOME=/tmp
+ENV TZ=UTC
```

Comment:
This was interesting.  DuckDB expects some OS level env vars to be set.  The AWS Lambda is mostly a "normal" environment for code to run, but sometimes run into things like this.

Setting `HOME` and `TZ` (timezone) are sufficient to allow DuckDB to run in a deployed lambda.